_posts/2019-07-24-hackillinois-a-journey.md

```diff
@@ -0,0 +1,118 @@
+
+One example is the friendship between [open source mentor Pablo Aguiar](https://blog.hackillinois.org/spotlights/2018/08/14/mentor-spotlight-pablo-aguiar.html) and staff member Shreyas Mohan.  The staff reached out to him for the 2018 event, not knowing that he would be travelling to the midwest from abroad. Nonetheless, [Pablo](https://www.github.com/scorphus) was incredibly excited about the event and travelled for days to make it. After the event, Shreyas and Pablo remained in touch, establishing a close staff-mentor friendship. Pablo returned to the event in 2019, again travelling for days from abroad to make it.
+
+Throughout the combined 72 hours, Pablo has mentored many groups of students, from beginners in open source to experienced developers. Many from the 2018 event continued to work on their project, an example of the success of the open source hackathon model.
+
+## Make Waves: Introducing Beginners to Open Source
+
+2019’s event marked a transition from separate tracks into one submission process. The team further refined the open source focus of the event by emphasizing the importance of welcoming beginners into the open source community.
+
+![Attendees speak to mentors about their projects at mentor matching after the opening ceremony. Photo by Yasha Mostofi.](/assets/article_images/hackillinois-a-journey/2019-mentor-matching.jpg)
+
+The beginner-friendly focus was driven by the desire for HackIllinois to be many people’s first exposure to open source. The staff sought to introduce as many people as possible to the ideas and the community that revolved around open source software. Beginners to open source would be the ideal target for the event.
+
+Although the event welcomed just as many freshman as it did seniors, mentors and students could see the shift towards welcoming beginners to open source. While not everyone was able to walk out the door with a prize, many felt like they left with something more important: new knowledge they gained from the people they interacted with over the course of the weekend.
+
+## Explore. Connect. Build: What Lies Beyond?
+
+For six years, HackIllinois has sought to bring the brightest young minds from across the country together and transform their ideas into reality. Year after year, the staff strives to provide everything a hacker needs for a weekend so they can focus on building something amazing. From the small details like hospitality to the overarching elements like our unique open source structure, the foundations on which we will build HackIllinois 2020 have been well-refined.
+
+While time may have passed, our commitment to promoting open source, creating a welcoming community to experts and beginners alike, and fostering a group of the most imaginative college students in the world has and will always remain the same.
```

Comment:
Maybe "While time has passed" since you can't really have an opinion on if time has passed or not.